test/DynamoCoreWpfTests/ConnectorViewModelTests.cs

```diff
@@ -26,6 +26,20 @@ public void ConnectorVisibilityForLegacyGraphTest()
             Assert.AreEqual(connectorViewModel.IsHidden, false);
         }
 
+        /// <summary>
+        /// Check to see a connector is visible after xml graph open
+        /// </summary>
+        [Test]
+        public void ConnectorVisibilityForLegacyXMLGraphTest()
+        {
+            Open(@"UI/UI_visual_test.dyn");
```

Comment:
if this test only makes sense if this .dyn is xml based, maybe we can check if the file is .xml as part of the test?

Reply:
Added XML check